frontend/src/features/research/add/AddResearchView.tsx

```diff
@@ -0,0 +1,70 @@
+import clsx from 'classnames';
+import { FilterProvider } from 'components/maps/providers/FIlterProvider';
+import {
+  SelectedPropertyContext,
+  SelectedPropertyContextProvider,
+} from 'components/maps/providers/SelectedPropertyContext';
+import { IProperty } from 'interfaces';
+import { noop } from 'lodash';
+import { useState } from 'react';
+import styled from 'styled-components';
+
+import Map from '../../../components/maps/leaflet/Map';
+import AddResearchSideBar from './AddResearchSideBar';
+
+/** rough center of bc Itcha Ilgachuz Provincial Park */
+const defaultLatLng = {
+  lat: 52.81604319154934,
+  lng: -124.67285156250001,
+};
+
+interface ResearchMapViewProps {
+  showParcelBoundaries?: boolean;
+  onMarkerPopupClosed?: (obj: IProperty) => void;
+}
+
+const ResearchMapView: React.FC<ResearchMapViewProps> = props => {
```

Comment:
The only problem I see with this is that the two map instances don't share state, so if we ever need to navigate between map instances the state will be reset. Not a problem at the moment but could be in the future.